src/envoy/mixer/README.md

```diff
@@ -131,7 +131,23 @@ By default, the mixer filter only forwards attributes and does not call mixer se
    }
 ```
 
-This route opaque config reverts the behavior by sending requests to mixer server but not forwarding any attributes.
+Above route opaque config reverts the behavior by sending requests to mixer server but not forwarding any attributes.
+
+Mixer attributes and forward attributes can be set per-route in the route opaque config.
+
+```
+ "routes": [
+   {
+     "timeout_ms": 0,
+     "prefix": "/",
+     "cluster": "service1",
+     "opaque_config": {
+      "mixer_attributes.key1": "value1",
```

Comment:
Envoy only supports flag name value pairs in opaque config

https://github.com/lyft/envoy/blob/master/include/envoy/router/router.h#L250